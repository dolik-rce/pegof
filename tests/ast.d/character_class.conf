input ast.d/character_class.peg
ast
