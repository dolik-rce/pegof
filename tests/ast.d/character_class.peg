X <- [^0-9] [^\t\v\n] [\x45-\x62] [\u0660-\u0669]
