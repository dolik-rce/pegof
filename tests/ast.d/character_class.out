GRAMMAR
  RULE X
    ALTERNATION
      SEQ
        TERM
          CHAR_CLASS 0-9
        TERM
          CHAR_CLASS \t\v\n
        TERM
          CHAR_CLASS E-b
        TERM
          CHAR_CLASS \u0660-\u0669
